aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -1190,6 +1310,47 @@ module aptos_framework::delegation_pool {
         });
     }
 
+    /// Set delegated voter of staker when initializing a delegation pool from a staking contract. This change applies
+    /// instantly without waiting a lockup cycle.
```

Comment:
Why is it important to change it instantly?

Reply:
this was a requirement from the specs